sample-app/src/main/java/com/joinforage/android/example/ui/pos/network/PosApiService.kt

```diff
@@ -30,6 +31,19 @@ interface PosApiService {
         @Body payment: PosPaymentRequest
     ): PaymentResponse
 
+    @POST("api/payments/{paymentRef}/void/")
+    suspend fun voidPayment(
+        @Header("Idempotency-Key") idempotencyKey: String,
+        @Path("paymentRef") paymentRef: String
+    ): PaymentResponse
+
+    @POST("api/payments/{paymentRef}/refunds/{refundRef}/void/")
+    suspend fun voidRefund(
+        @Header("Idempotency-Key") idempotencyKey: String,
+        @Path("paymentRef") paymentRef: String,
+        @Path("refundRef") refundRef: String
+    ): PaymentResponse
+
```

Comment:
I am _seriously_ digging how convenient it is to declare typed http apis with `retrofit`. Nice!